README.md

```diff
@@ -8,7 +8,7 @@ This is a [fonk](https://github.com/Lemoncode/fonk) microlibrary that brings val
 
 // TODO: Update description and example.
 
-- Validate if a field of a form ....
+- Validate if a field of a form is marked as a BIC code format. 
```

Comment:
Validate if a field form is a valid BIC (Bank International Code a.k.a SWIFT) code.

Reply:
Ok, done